packages/blocks/src/api/parser/index.js

```diff
@@ -81,11 +82,12 @@ function convertLegacyBlocks( rawBlock ) {
  * Normalize the raw block by applying the fallback block name if none given,
  * sanitize the parsed HTML...
  *
- * @param {WPRawBlock} rawBlock The raw block object.
+ * @param {WPRawBlock}   rawBlock The raw block object.
+ * @param {ParseOptions} options  Extra options for handling block parsing.
```

Comment:
Shouldn't the param be optional?

Reply:
Yep, I've updated that, along with making the boolean options for `ParseOptions` optional too.